docker-stack.yml

```diff
@@ -0,0 +1,83 @@
+version: '3.3'
```

Comment:
Not sure that we're using any 3.3 specific features here, might want to use 3 instead.

Reply:
Seems like it is, Travis just failed with
```
$ docker stack deploy -c ./docker-stack.yml elk
configs Additional property configs is not allowed
```
Reverting.

Reply:
Found this in the documentation @ https://docs.docker.com/compose/compose-file/#short-syntax:
> Note: `config` definitions are only supported in version 3.3 and higher of the compose file format.

Reply:
Yup, we should stay on 3.3.